core/sql/generator/GenExpGenerator.h

```diff
@@ -1086,45 +1073,7 @@ class ExpGenerator : public NABasicObject
 		     const SearchKey * searchKey,
 		     const MdamKey * mdamKey,
 		     const NABoolean reverseScan,
-		     unsigned short keytag,
-		     const ExpTupleDesc::TupleDataFormat tf,
-
-		     // the next few parameters are here
-		     // as part of a horrible kludge for
-		     // the PartitionAccess::codeGen()
-		     // method, which lacks a SearchKey
-		     // object and therefore exposes
-		     // things like the exclusion
-		     // expressions; with luck, later work
-		     // in the Optimizer will result in a
-		     // much cleaner interface
-		     const NABoolean useTheHorribleKludge = FALSE,
```

Comment:
Glad to see this finally removed/replaced. I remember putting this in around 1996 to fix some issue with the interaction of MDAM and partition access.